deno/rest/common.ts

```diff
@@ -177,6 +177,8 @@ export enum RESTJSONErrorCodes {
 
 	ServerNeedsMoreBoostsToPerformThisAction = 50101,
 
+	InvalidJSON = 50109,
```

Comment:
Should we keep the shorter version or elongate it?

Reply:
Couldn't hurt to be a little more descriptive. `RequestBodyInvalidJSON`?

Reply:
Hmmm..nah, just elongate it like the docs have it